Source/WebKit/UIProcess/Cocoa/VideoPresentationManagerProxy.messages.in

```diff
@@ -21,11 +21,9 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #if ENABLE(VIDEO_PRESENTATION_MODE)
-[
-    ExceptionForEnabledBy,
-    DispatchedFrom=WebContent,
-    DispatchedTo=UI
```

Comment:
Just confirming that you intended to remove `DispatchedFrom` and `DispatchedTo`.